src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -0,0 +1,264 @@
+    col_name = get_col_name("melt", outlier_type)
+    outlier_melt_df = read_outlier(filename_melt, col_name)
+    dict_melt = dict(list(zip(outlier_melt_df[ID_COL], outlier_melt_df[col_name])))
+
+    # Wham:
+    col_name = get_col_name("wham", outlier_type)
+    outlier_wham_df = read_outlier(filename_wham, col_name, 11)
+    dict_wham = dict(list(zip(outlier_wham_df[ID_COL], outlier_wham_df[col_name])))
+
+    # merging all the dictionaries
+    outlier_dicts = [dict_manta, dict_melt, dict_wham]
+    merged_dicts = Counter()
+    for counted in outlier_dicts:
+        merged_dicts.update(counted)
+    all_outliers = dict(merged_dicts)
+    all_outliers_df = pd.DataFrame.from_dict(all_outliers, orient="index").reset_index()  # index_col=None
+    all_outliers_df.columns = [ID_COL, outlier_type + "_overall_outliers"]
+    return all_outliers_df
+
+
+def merge_evidence_qc_table(
+        filename_estimated_cn: str,
+        filename_mediancov: str,
+        filename_wgd: str,
+        filename_cnv_qvalues: str,
+        filename_high_manta: str,
+        filename_high_melt: str,
+        filename_high_wham: str,
+        filename_low_manta: str,
+        filename_low_melt: str,
+        filename_low_wham: str,
+        output_prefix: str) -> str:
+    """
+    Reads the provided TSV files (tab-delimited) and merges all the information in one table
+    serialized to the given output filename.
+    """
+    df_ploidy = read_ploidy(filename_estimated_cn)
+    df_bincov_median = read_bincov_median(filename_mediancov)
+    df_wgd_scores = read_wgd_scores(filename_wgd)
+    df_non_diploid = read_non_diploid(filename_cnv_qvalues)
+    df_manta_high_outlier = read_outlier(filename_high_manta, get_col_name("manta", "high"))
+    df_melt_high_outlier = read_outlier(filename_high_melt, get_col_name("melt", "high"))
+    df_wham_high_outlier = read_outlier(filename_high_wham, get_col_name("wham", "high"), 11)
+    df_total_high_outliers = read_all_outlier(filename_high_manta, filename_high_melt, filename_high_wham, "high")
+    df_manta_low_outlier = read_outlier(filename_low_manta, get_col_name("manta", "low"))
+    df_melt_low_outlier = read_outlier(filename_low_melt, get_col_name("melt", "low"))
+    df_wham_low_outlier = read_outlier(filename_low_wham, get_col_name("wham", "low"), 11)
+    df_total_low_outliers = read_all_outlier(filename_low_manta, filename_low_melt, filename_low_wham, "low")
+
+    # all data frames
+    dfs = [df_ploidy, df_bincov_median, df_wgd_scores, df_non_diploid, df_manta_high_outlier,
+           df_melt_high_outlier, df_wham_high_outlier, df_total_high_outliers,
+           df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, df_total_low_outliers]
+    output_df = reduce(
+        lambda left, right: pd.merge(left, right, on=ID_COL, how="outer"), dfs
+    ).fillna(0)  # "none"
+
+    # save the file
+    return output_df.to_csv(f"{output_prefix}evidence_qc_table.tsv", sep="\t", header=True, index=False)
```

Comment:
Let's just make the prefix required so we can add a delimiter here
```suggestion
    return output_df.to_csv(f"{output_prefix}.evidence_qc_table.tsv", sep="\t", header=True, index=False)
```